### PR TITLE
Feature/add sds input

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: extension/src/popup/views/UnlockAccount.tsx [hunk 1]
```diff
@@ -108,15 +100,19 @@ export const UnlockAccount = () => {
...
+                      error={authError}
+                      placeholder="Enter Password"
+                      type="password"
+                      {...field}
+                    />
+                  )}
+                </Field>
               </FormRow>
-              <ApiErrorMessage error={authError} />
               <ButtonRowEl>
```
Comment (piyalbasu) [inline extension/src/popup/views/UnlockAccount.tsx:116]: This should be a pretty good pattern to follow when implementing an SDS Input that's connected to Formik

Comment (piyalbasu) [reply extension/src/popup/views/UnlockAccount.tsx:116]: We can remove <ApiErrorMessage /> as SDS Input will handle displaying the error 🎉

**Review by acharb — APPROVED** [review]
lgtm!

**Review by quietbits — APPROVED** [review]
:tada:

**Pull request merged**